tools/serverpod_cli/lib/src/update_prompt/prompt_to_update.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:pub_semver/pub_semver.dart';
+import 'package:serverpod_cli/src/util/latest_cli_version.dart';
+
+/// Check current Serverpod CLI version and prompt user to update if needed
+Future<void> promptToUpdateIfNeeded(Version currentVersion) async {
+  var latestVersion = await tryFetchLatestValidCliVersion();
+  if (latestVersion == null) return;
+
+  if (currentVersion < latestVersion) {
+    _printPrompt(latestVersion);
+  }
+}
+
+void _printPrompt(Version version) {
+  print('');
+  print(_promptMessage(version));
+  print('');
+}
+
+String _promptMessage(Version version) {
```

Comment:
Using the version of the print message that @dmytro-virych created. We will revisit the output style once we have a centralised log service.